src/config/config.ts

```diff
@@ -390,6 +399,23 @@ export class Config implements IConfig {
     }
   }
 
+  public async loadThemes(): Promise<{
+    file: string
+    activeTheme: Theme | undefined
+    themes: Themes | undefined
+  }> {
+    const themesFile = this.pjson.oclif.themesFile
+      ? resolve(this.root, this.pjson.oclif.themesFile)
+      : resolve(this.configDir, 'themes.json')
+    const themes = await safeReadJson<Themes>(themesFile)
+    const activeTheme = themes ? parseTheme(themes) : undefined
+    return {
+      activeTheme,
+      file: themesFile,
+      themes,
+    }
+  }
+
```

Comment:
I don't understand why loading all themes in memory if only one is used at a time

Reply:
It does not answer the question. What is the use case to have all themes in memory?